packages/transformers/webext/src/WebExtTransformer.js

```diff
@@ -0,0 +1,261 @@
+        isEntry: true,
+      });
+    }
+  }
+  if (program.content_scripts) {
+    for (let i = 0; i < program.content_scripts.length; ++i) {
+      const sc = program.content_scripts[i];
+      for (const k of ['css', 'js']) {
+        const assets = sc[k] || [];
+        for (let j = 0; j < assets.length; ++j) {
+          assets[j] = asset.addURLDependency(assets[j], {
+            isEntry: k == 'js',
+            loc: {
+              filePath,
+              ...getJSONSourceLocation(
+                ptrs[`/content_scripts/${i}/${k}/${j}`],
+                'value',
+              ),
+            },
+          });
+        }
+      }
+    }
+  }
+  if (program.dictionaries) {
+    for (const dict in program.dictionaries) {
+      const sourceLoc = getJSONSourceLocation(
+        ptrs[`/dictionaries/${dict}`],
+        'value',
+      );
+      const loc = {
+        filePath,
+        ...sourceLoc,
+      };
+      const dictFile = program.dictionaries[dict];
+      if (extname(dictFile) != '.dic') {
+        throw new ThrowableDiagnostic({
+          diagnostic: [
+            {
+              message: 'Dictionaries must be .dic files',
+              origin: '@parcel/transformer-webext',
+              filePath,
+              codeFrame: {
+                codeHighlights: [((sourceLoc: any): DiagnosticCodeHighlight)],
+              },
+            },
+          ],
+        });
+      }
+      program.dictionaries[dict] = asset.addURLDependency(dictFile, {
+        isEntry: true,
+        loc,
+      });
+      asset.addURLDependency(dictFile.slice(0, -4) + '.aff', {
+        isEntry: true,
+        loc,
+      });
+    }
+  }
+  if (program.web_accessible_resources) {
+    for (let i = 0; i < program.web_accessible_resources.length; ++i) {
+      // TODO: this doesn't support Parcel resolution
+      const globQuery = join(
+        dirname(filePath),
+        program.web_accessible_resources[i],
+      );
+      for (const fp of await glob(globQuery, fs, {})) {
+        asset.addURLDependency(relative(dirname(filePath), fp), {
+          isEntry: true,
+          loc: {
+            filePath,
+            ...getJSONSourceLocation(ptrs[`/web_accessible_resources/${i}`]),
+          },
+        });
+      }
+    }
+  }
+  for (let loc of DEP_LOCS) {
+    let isEntry: boolean = false;
+    if (typeof loc[loc.length - 1] == 'boolean') {
+      isEntry = ((loc[loc.length - 1]: any): boolean);
+      loc = loc.slice(0, -1);
+    }
+    const locStr = '/' + loc.join('/');
+    let obj: any = program;
+    for (let i = 0; i < loc.length - 1; ++i) {
+      obj = obj[loc[i]];
+      if (!obj) break;
+    }
+    if (!obj) continue;
+    const parent = obj,
+      lloc = loc[loc.length - 1];
+    obj = obj[lloc];
+    if (!obj) continue;
+    if (typeof obj == 'string')
+      parent[lloc] = asset.addURLDependency(
+        // TODO: not this, for sure
+        (extname(obj) == '.json' ? 'raw:' : '') + obj,
```

Comment:
Is `raw` needed for other fields apart from `web_accessible_resources`? Not really a fan of this but we'll have to work around this for now...

Reply:
This is needed for anything with JSON because the default JSON transformer transforms JSON to javascript. In terms of WebExt, this is needed for that and `storage.managed_schema`.